image_generation/stable_diffusion_1_5/cpp/README.md

```diff
@@ -100,16 +100,7 @@ Negative prompt: (empty, here couldn't use OV tokenizer, check the issues for de
 
 Read the numpy latent instead of C++ std lib for the alignment with Python pipeline
 
-* Generate image without lora `./stable_diffusion -r`
-
-![image](https://github.com/intel-sandbox/OV_SD_CPP/assets/102195992/66047d66-08a3-4272-abdc-7999d752eea0)
```

Comment:
maybe we can upload them to this repo? 
It's convenient for users to see expected result and understand their setup is valid